doc/travis-ci.md

```diff
@@ -43,7 +43,39 @@ That's it! Lets look at the important lines.
 * `devDependencies` simply tells [npm](https://npmjs.org) what to install before running tests. Travis will actually do this for us via the `npm install` command in our project directory.
 * `scripts` will be run by Travis via the `npm test` command to actually run our tests. Tests will consider passing if this command returns successfully. You can actually test it locally by running `npm test` in your terminal.
 
-### 5. Final step
+### 5. Code coverage (optional)
+
+Code coverage is disabled by default so this step is optional. You opt-in by using the `--coverage` cli option. This will collect code coverage per browser into `.nyc-output/airtap-<hash>.json` in Istanbul 1.0 format. Later you can generate reports by using e.g. `nyc report`, which takes care of merging files from multiple browsers.
+
+In local mode, `airtap` writes to `.nyc-report/airtap-local.json`, so that no cleanup is necessary between repeated runs, e.g. if you refresh the page.
+
+A typical setup in Travis could look like:
+
+```json
+{
+  "scripts": {
+    "test": "airtap --sauce-connect --loopback airtap.local --coverage test.js"
+  }
+}
+```
+
+Additionally, you can add hook in Travis to post the results to `coveralls`:
```

Comment:
"add hook in Travis" => perhaps just "configure Travis".

I'm not sure I was right to call this a "hook" in #194, might be "step" or "stage".